src/components/main-form.js

```diff
@@ -519,19 +546,93 @@ class MainForm extends React.Component {
                   and your return on investment lower. We factor all of this
                   automatically so the final result you see on your right is not
                   the actual money you will have gained/lost in{' '}
-                  {this.state.longerOption.term} years, but rather, the money
-                  you will have gained/lost in {this.state.longerOption.term}{' '}
-                  years adjusted to today’s purchasing power. If you’re curious
-                  about what the actual number would be, give inflation a value
-                  of 0%. But remember that everything will be a lot more
-                  expensive in {this.state.longerOption.term} years, and that is
-                  why we need to include inflation.
+                  {longerOption.term} years, but rather, the money you will have
+                  gained/lost in {longerOption.term} years adjusted to today’s
+                  purchasing power. If you’re curious about what the actual
+                  number would be, give inflation a value of 0%. But remember
+                  that everything will be a lot more expensive in{' '}
+                  {longerOption.term} years, and that is why we need to include
+                  inflation.
                 </Accordion>
               </FormGroup>
             </Form>
           </Card>
         </Col>
         <Col xs='8'>
+          <Summary className='p-4 mr-5'>
```

Comment:
sug: Perhaps this could just be `Row`?

Reply:
~For the spacing, you can apply `no-gutters` to this row and it will align:~

https://getbootstrap.com/docs/4.1/layout/grid/#no-gutters

Ah, looking at the alignment issues, it seems that padding is the explanation not having padding is what's causing a misalignment

Reply:
> sug: Perhaps this could just be `Row`?

it is Row! I just styled it and gave it a more intuitive name.  What exactly should I do about spacing? I'm not sure if I follow

Reply:
Ah, it's related to styled components we talked about. The `Summary` seemed to just be a parent that wraps more specific child styles. After playing w/ it, I think this could just be Row w/o any child CSS in it